swift-sdk/IterableConfig.swift

```diff
@@ -35,16 +35,27 @@ import Foundation
 }
 
 /**
- * This protocol allows you to override default behavior when new inApps arrive.
+ * This protocol allows you to override default behavior when new in-app messages arrive.
  */
 @objc public protocol IterableInAppDelegate : class {
     /**
-     * This method is called when new inApp message is available.
+     * This method is called when inbox is ready. This is called right after app is launched and inbox messages
+     * are loaded from storage.
+     * - parameter messages: Array of `IterableInAppMessage` object containing information regarding inbox messages
+     */
+    @objc(onInboxReady:) func onInboxReady(messages: [IterableInAppMessage])
```

Comment:
Rename to `onReady`? This also applies to in-app messages.

Reply:
Actually we might not even need this if we have `onInboxChange`. It will run the first time we load the messages into memory.